### DISCUSSION TITLE
should jest config from "package.json" be moved to a dedicated "jest.config.js" file?

### DISCUSSION THREAD

**Discussion by Adnan-Bacic** [react-native-community/discussions-and-proposals#583] [Ideas]
according to: https://jestjs.io/docs/configuration

> It is recommended to define the configuration in a dedicated JavaScript, TypeScript or JSON file. The file will be discovered automatically, if it is named jest.config.js|ts|mjs|cjs|json.

currently, new react-native projects have the jest config in `package.json`. i suggest this is moved to a dedicated `jest.config.js` file.

**Comment by kelset** [comment]
makes sense to me 👍

**Comment by Adnan-Bacic** [comment]
pr merged: https://github.com/facebook/react-native/pull/35856
